src/routes/+page.ts

```diff
@@ -0,0 +1,123 @@
+import type { PageLoad } from "./$types";
+import { BackendController } from "$lib/controller/backend-controller";
+import {
+    type Paper,
+    type Project,
+    type ProjectMetadata,
+    type StageEntry,
+    type User,
+} from "$lib/model/backend";
+import { calculateStageProgress } from "$lib/utils/statistics-helper";
+
+// import type { Project, ProjectMetadata } from "$lib/model/backend";
+
+async function requestProjectMetadata(project: Project): Promise<ProjectMetadata> {
+    const members: User[] = await BackendController.getInstance().project(project.id).getMembers();
+    const currentStage: number = await BackendController.getInstance()
+        .project(project.id)
+        .getStageCount();
+    const allPapersInCurrentStage: StageEntry[] = await BackendController.getInstance()
+        .project(project.id)
+        .stage(currentStage)
+        .getPapers();
```

Comment:
You can first create the project controller and reuse it for the subsequent requests.
```suggestion
    const projectController = BackendController.getInstance().project(project.id);
    const members: User[] = await projectController.getMembers();
    const currentStage: number = await projectController.getStageCount();
    const allPapersInCurrentStage: StageEntry[] = await projectController
        .stage(currentStage)
        .getPapers();
```

Reply:
Furthermore, I got this error message:

 Loading http://localhost:3000/projects/2/stages/0/papers using `window.fetch`. For best results, use the `fetch` that is passed to your `load` function: https://svelte.dev/docs/kit/load#making-fetch-requests

Did you try anything with this? Let's investigate this further in a separate issue.

Reply:
Thank you for this hint, I try to remember it for the future 👍.

No i dient not try anything with that, but I also noticed this warning and read the documentation regarding the fetch. But as we will switch to gRPC soon, we will delete the fetch calls anyway, so I did not see any reason, why to investigate further. Do you want me to investigate?

Reply:
No, let's see what happens when we use gRPC.